src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -39,7 +39,7 @@ def dialog_content
         VBox(
           alternative_information,
           create_alternatives_table,
-          RichText(Id(:slaves), _("Please select an alternative to view his slaves.")),
+          RichText(Id(:slaves), "<pre>#{@alternative.choices.first.slaves}</pre>"),
```

Comment:
or as I said, let it empty and after dialog creation select currently selected item and then call this handler to show proper aliases